rules/core/block-no-empty/README.md

```diff
@@ -0,0 +1,31 @@
+#### 📍 block-no-empty
+
+Disallow empty blocks.
+
+##### 🧟 Example of incorrect code for this rule:
```

Comment:
convert to new file structure. please see https://github.com/netsells/stylelint-config-netsells/pull/17 for an example